lms/djangoapps/certificates/models.py

```diff
@@ -59,6 +59,7 @@
 from django.dispatch import receiver
 from django.conf import settings
 from django.utils.translation import ugettext_lazy as _
+from django.utils.translation import ungettext_lazy
```

Comment:
what is reason for importing `ungettext_lazy` when it is already imported in the line above?